### PR TITLE
Issue 53 : GiveofferController/compile output

### PR DESCRIPTION
**Pull request by Meenal2020**

### CODE REVIEW

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  private OfferDTO offerDTO;
+
+  @BeforeEach
+  public void setUpTest() {
+    offerDTO = giveofferController.call(OfferDTOTestData.get());
+  }
+
+  @Test
+  @DisplayName("OfferDTO returned successfully")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: The DisplayName annotation should be a full English sentence understandable by the end user, as it goes to the end-user documentation. E.g. "retuns an offer"

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  public void testCompileOutput() {
+
+    assertEquals(
+        OfferDTOTestData.get(), offerDTO
+    );
+  }
+
+  @Test
+  @DisplayName("Valid price tested successfully")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: see above. "The price of the offer is returned" is a good solution here.

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  public void testPrice() {
+    assertEquals(
+        OfferDTOTestData.VALID_PRICE,
+        offerDTO.getPrice()
+    );
+  }
+
+  @Test
+  @DisplayName("Id tested successfully")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: see above

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  public void testId() {
+    assertEquals(
+        OfferDTOTestData.get().getId(),
+        offerDTO.getId()
+    );
+  }
+
+  @Test
+  @DisplayName("WorRequest tested successfully")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: see above

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java (hunk 2 above)
Comment (Meenal2020) [reply src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: changes done and code committed

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java (hunk 3 above)
Comment (Meenal2020) [reply src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: done

File: src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java (hunk 4 above)
Comment (Meenal2020) [reply src/test/java/com/kodekonveyor/work_request/offer/GiveofferControllerCompileOutputTest.java]: done

**Pull request merged**